habitat-lab/habitat/utils/visualizations/utils.py

```diff
@@ -291,3 +289,76 @@ def append_text_to_image(image: np.ndarray, text: str):
     text_image = blank_image[0 : y + 10, 0:w]
     final = np.concatenate((image, text_image), axis=0)
     return final
+
+
+def append_text_to_image(
+    image: np.ndarray, text: List[str], font_size: float = 0.5
+):
+    r"""Appends lines of text on top of an image. First this will render to the
+    left-hand side of the image, once that column is full, it will render to
+    the right hand-side of the image.
+    :param image: the image to put text underneath
+    :param text: The list of strings which will be rendered, separated by new lines.
+    :returns: A new image with text inserted underneath the input image
+    """
+    h, w, c = image.shape
+    font_thickness = 1
+    font = cv2.FONT_HERSHEY_SIMPLEX
+
+    y = 0
+    left_aligned = True
+    for line in text:
+        textsize = cv2.getTextSize(line, font, font_size, font_thickness)[0]
+        y += textsize[1] + 10
+        if y > h:
+            left_aligned = False
+            y = textsize[1] + 10
+
+        if left_aligned:
+            x = 10
+        else:
+            x = w - (textsize[0] + 10)
+
+        cv2.putText(
+            image,
+            line,
+            (x, y),
+            font,
+            font_size,
+            (0, 0, 0),
+            font_thickness * 2,
+            lineType=cv2.LINE_AA,
+        )
+
+        cv2.putText(
```

Comment:
Why is putText called twice here ?

Reply:
I just moved a function to new location.